src/Adapter/CMS/Page/CommandHandler/AddCmsPageHandler.php

```diff
@@ -77,13 +78,17 @@ public function handle(AddCmsPageCommand $command)
      * @return CMS
      *
      * @throws PrestaShopException
+     * @throws CmsPageCategoryException
```

Comment:
In fact, this function doesn't throws exception, otherwise if you want to add all exceptions, you must add all database exception, Exception and others.